index.js

```diff
@@ -205,10 +206,28 @@ function compileAsync(expression) {
   return evaluateAsync.bind(null, jsep(expression));
 }
 
+// Added functions to inject Custom Unary Operators (and override existing ones)
+function addCustomUnOps(customUnops){
+  _.forOwn(customUnops, (value, key) => {
+		jsep.addUnaryOp(key);
+		_.merge(unops, {[key]: value});
+	});
+}
+
+// Added functions to inject Custom Binary Operators (and override existing ones)
+function addCustomBinOps(customBinops){
+  _.forOwn(customBinops, (value, key) => {
+		jsep.addUnaryOp(key);
+		_.merge(binops, {[key]: value});
+	});
+}
+
 module.exports = {
   parse: jsep,
   eval: evaluate,
   evalAsync: evaluateAsync,
   compile: compile,
-  compileAsync: compileAsync
+  compileAsync: compileAsync,
+  addCustomUnOps: addCustomUnOps,
+  addCustomBinOps: addCustomBinOps
```

Comment:
I'd prefer to match the name's used by jsep (https://github.com/soney/jsep#custom-operators) for these methods:

```js
addBinaryOp('^', (a, b) => { ... });
addUnaryOp('@', (a) => { ... });
```